cypress/integration/change_profile_picture.js

```diff
@@ -36,6 +36,8 @@ describe("Change Profile Picture", () => {
             })
             .get(".photo-edit-button")
             .click()
+            cy.get(".sun > img")
```

Comment:
Why was this added to the test?

Reply:
It was added to the test because the button now redirects to the dashboard. This is because of the conditional rendering needed inside the component so it can be used by Gallery too!